src/parser/compile.c

```diff
@@ -1052,21 +1102,22 @@ remove_appended_trigger_info (char *msg, int with_evaluate)
  *   class_op(in): class name to resolve name1 and name2 to
  *   name1(in): name to resolve
  *   name2(in): name to resolve
+ *   new_trigger_stmt(in):
+ *   with_evaluate(in):
  *
  */
 
 PT_NODE *
 pt_compile_trigger_stmt (PARSER_CONTEXT * parser, const char *trigger_stmt, DB_OBJECT * class_op, const char *name1,
-			 const char *name2)
+			 const char *name2, char **new_trigger_stmt, int with_evaluate)
 {
   char *stmt_str = NULL;
   const char *class_name;
   PT_NODE **statement_p, *statement;
   int is_update_object;
   PT_NODE *err_node;
-  int with_evaluate;
 
-  assert (parser != NULL);
+  assert (parser != NULL || !*new_trigger_stmt);
```

Comment:
```suggestion
  assert (parser != NULL || (new_trigger_stmt != NULL && *new_trigger_stmt == NULL));
```

위 assert의 의미일까요?? debug 모드에서 new_trigger_stmt가 NULL인 경우 assert에서 new_trigger_stmt를 dereference 하다가 segfault가 발생할 수 있을 것 같습니다.

Reply:
코드 리뷰 감사합니다.

리뷰 내용을 참고하여 "parser와 new_trigger_stmt가 NULL이 아닌 경우"를 확인하도록 아래와 같이 assert 조건을 변경했습니다.
`assert (parser != NULL && new_trigger_stmt != NULL)`